src/pybind/mgr/nfs/export.py

```diff
@@ -656,6 +661,22 @@ def create_cephfs_export(self,
                              access_type: str,
                              clients: list = [],
                              sectype: Optional[List[str]] = None) -> Dict[str, Any]:
+
+        with open_filesystem(self.fs_client, fs_name) as fs_handle:
+            try:
+                stx = fs_handle.statx(path.encode('utf-8'),
+                                      cephfs.CEPH_STATX_MODE |
+                                      cephfs.CEPH_STATX_UID |
```

Comment:
nit: are flags for uid, gid, atime, mtime and size required here? the check is only to ensure the file is a directory.

Reply:
you're right, i added them while debugging the make check weird failure and then missed to remove then, i've removed them now. thanks!